static_src/actions/service_actions.js

```diff
@@ -146,10 +147,13 @@ const serviceActions = {
     return Promise.resolve(serviceInstance);
   },
 
-  errorCreateInstance(err) {
+  errorCreateInstance(error) {
+    const { response } = error;
+    const safeError = (response && response.data) || { code: 500 };
```

Comment:
Using this construct in JS, if the first argument evaluates to `true`, and the second argument is also true, the second argument will be executed. In this instance, executing means storing the value of the second argument in a variable, but you could, for example,  pass a function as the second argument.

If the second argument is also false, the fallback will be executed instead.

I concede that this might look a little confusing but it is a pretty common in JS.

If I misunderstood the question, and you already knew this, sorry 😬 !